<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="cpj" name="Login" tests="5" failures="1" timestamp="30 Aug 2017 09:22:31 GMT" time="58.818" errors="0">
  <testcase name="loginErrorUser" time="39.372" classname="com.yiibai.primera.testng.cases.LoginTest"/>
  <testcase name="loginErrorNum" time="4.276" classname="com.yiibai.primera.testng.cases.LoginTest"/>
  <testcase name="loginChinese" time="4.515" classname="com.yiibai.primera.testng.cases.LoginTest"/>
  <testcase name="loginEmpty" time="4.218" classname="com.yiibai.primera.testng.cases.LoginTest"/>
  <testcase name="loginConfim" time="4.654" classname="com.yiibai.primera.testng.cases.LoginTest">
    <failure type="java.lang.AssertionError" message="帐号密码对的情况登录 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 帐号密码对的情况登录 expected [true] but found [false]
	at com.yiibai.primera.testng.cases.LoginTest.loginConfim(LoginTest.java:76)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginConfim -->
</testsuite> <!-- Login -->
